releases/cosmwasm/2025-02-XRPL-v1.0.0.md

```diff
@@ -0,0 +1,364 @@
+# XRPL Amplifier
+
+|                | **Owner**                        |
+| -------------- | -------------------------------- |
+| **Created By** | @k4m4 <nikolas@commonprefix.com> |
+| **Deployment** | @k4m4 <nikolas@commonprefix.com> |
+
+| **Network**          | **Deployment Status** | **Date**   |
+| -------------------- | --------------------- | ---------- |
+| **Devnet Amplifier** | Deployed              | 2025-01-29 |
+| **Stagenet**         | -                     | TBD        |
+| **Testnet**          | -                     | TBD        |
+| **Mainnet**          | -                     | TBD        |
+
+- [Amplifier Releases](https://github.com/commonprefix/axelar-amplifier/releases)
+- [XRPLVotingVerifier v1.1.0](https://github.com/commonprefix/axelar-amplifier/releases/tag/xrpl-voting-verifier-v1.1.0)
+- [XRPLGateway v1.1.1](https://github.com/commonprefix/axelar-amplifier/releases/tag/xrpl-gateway-v1.1.1)
+- [XRPLMultisigProver v1.1.1](https://github.com/commonprefix/axelar-amplifier/releases/tag/xrpl-multisig-prover-v1.1.1)
+
+## Background
+
+These are the instructions for deploying Amplifier contracts for the XRPL connection.
+
+### Pre-requisites
+
+Ensure that the [XRPL multisig account](../xrpl/2025-02-XRPL-v1.0.0.md) is created and configured first, as `XRPLVotingVerifier` needs the XRPL multisig account's address as `sourceGatewayAddress` and the `XRPLGateway` and `XRPLMultisigProver` contracts need it as `xrplMultisigAddress`.
+
+## Deployment
+
+- Create an `.env` config. `CHAIN` should be set to `xrpl` for mainnet, and `xrpl-2025-q1` for all other networks.
+
+```yaml
+MNEMONIC=xyz
+ENV=xyz
+CHAIN=xyz
+```
+
+- Confirm `XRPLVotingVerifier(v1.1.0)`, `XRPLGateway(v1.1.1)` and `XRPLMultisigProver(v1.1.1)` contracts are already stored in `$ENV.json`
```

Comment:
is there an example for this json file ?

Reply:
There's just `devnet-amplifier.json` in `axelar-chains-config/info`, but it's missing `storeCodeProposalId` and `storeCodeProposalCodeHash`, since we didn't need to submit a proposal for devnet. The process should be the same as for the usual CosmWasm contracts, just `VotingVerifier` -> `XrplVotingVerifier`, `Gateway` -> `XrplGateway`, and `MultisigProver` -> `XrplMultisigProver`.

Reply:
oh I thought you meant store the contract artifacts in a json file somehow.

Reply:
Nope, we copied this text from `releases/cosmwasm/2025-01-Stellar-GMP-v1.0.0.md`.